inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongClientImpl.java

```diff
@@ -129,11 +129,11 @@ public InlongGroup forGroup(InlongGroupConf groupConf) {
         List<InlongGroupListResponse> groupListResponses = pageInfoResponse.getData().getList();
         Map<String, InlongGroupState> groupStateMap = Maps.newHashMap();
         groupListResponses.stream().forEach(groupListResponse -> {
-            String groupId = groupListResponse.getInlongGroupId();
+            String groupName = groupListResponse.getName();
```

Comment:
We will optimize the configuration of inlong_group_id and name in future versions, and it is currently not recommended to do such optimization.